pom.xml

```diff
@@ -251,6 +251,10 @@
 				<enabled>true</enabled>
 			</snapshots>
 		</repository>
+		<repository>
+			<id>mvnrepository.com</id>
```

Comment:
Unneeded repository. Even if you used Maven to get common-lang3 (you shouldn't in that case), it's already available in Maven Central